README.md

```diff
@@ -1,48 +1,44 @@
-[![npm version](https://img.shields.io/npm/v/@gnosis.pm/gp-v1-ui.svg?style=flat)](https://npmjs.org/package/@gnosis.pm/gp-v1-ui 'View this project on npm')
-&nbsp;
-[![Build Status](https://travis-ci.org/gnosis/gp-v1-ui.svg?branch=develop)](https://travis-ci.org/gnosis/gp-v1-ui)
-&nbsp;
-[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1-ui/badge.svg?branch=master)](https://coveralls.io/github/gnosis/gp-v1-ui?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1/badge.svg?branch=master)](https://coveralls.io/github/gnosis/gp-v1?branch=master)
 
 Develop:
-&nbsp;
-[![Build Status](https://travis-ci.org/gnosis/gp-v1-ui.svg?branch=develop)](https://travis-ci.org/gnosis/gp-v1-ui)
-&nbsp;
-[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1-ui/badge.svg?branch=develop)](https://coveralls.io/github/gnosis/gp-v1-ui?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v1/badge.svg?branch=develop)](https://coveralls.io/github/gnosis/gp-v1?branch=develop)
 
-# Gnosis Protocol v1 Web
+# Gnosis Protocol UI
```

Comment:
I'd say keep `v1` in the header

Reply:
why? this is not V1 anymore, the default app now is the explorer

Reply:
Oh, that's what you mean.
I thought it was still indicating the former `dex-react` app.